src/bandersnatch/mirror.py

```diff
@@ -70,7 +74,11 @@ def __init__(
         digest_name=None,
         root_uri=None,
         keep_index_versions=0,
+        diff_file=None,
+        diff_append_epoch=False,
+        diff_full_path=None,
         flock_timeout=1,
+        diff_file_list=[],
```

Comment:
Please do not set an immutable default (sorry should have a lint for this). It's set / evaluated @ definition time, not on function call.

http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments

```
diff_file_list=None,
```